SQLite/Core/Connection.swift

```diff
@@ -22,6 +22,7 @@
 // THE SOFTWARE.
 //
 
+import Foundation
```

Comment:
Not as an optimization, more as a documentation of the dependencies of the file. But it's not commonly done in Swift/ObjC.